strapi/strapi
# Validate selected entries for bulk publish

### What does it do?

- Moves schema.js util higher up so it can be accessed be EditView and ListView
- creates a yup schema in the list view and validates selected entries against the contentType

### Why is it needed?

Describe the issue you are solving.

### How to test it?

- Add some validations in the CTB (required, min length, etc...)
- Go to the CM and create some drafts as valid and invalid entries
- Go to the list view and try to bulk publish
- A log should be in the console with the validIds that can be published and the errors for each id that didn't pass the validation
